### DISCUSSION TITLE
Server error

### DISCUSSION THREAD

**Discussion by Balakay420** [shedaniel/RoughlyEnoughItems#941] [Q & A] [answered]
I keep getting this error when trying to join my server, It's not crashing the server it just wont let me join.  I was wondering if you could help me
[logs_latest (4).log](https://github.com/shedaniel/RoughlyEnoughItems/files/8937331/logs_latest.4.log)
.

**Comment by timtornado** [comment, marked as answer]
dwayne-the-block-johnson has an error....
also 
 - Mod 'Omega Config GUI' (omega-config-gui) 1.2.3-1.18.1 recommends version 3.0.1 of modmenu, which is disabled for this environment (client/server only)!
	 - You should install version 3.0.1 of modmenu for the optimal experience.
	 
REI: Server failed to synchronize recipe data with the client! Please check the server console log for errors,
this is all in there bud you just gotta look
